streaming/aggregate/aggregate_test.go

```diff
@@ -0,0 +1,72 @@
+package aggregate
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cube2222/octosql"
+	"github.com/cube2222/octosql/streaming/storage"
+)
+
+const eps = 0.00000001
+
+func ExpectValue(t *testing.T, ctx context.Context, aggr Aggregate, tx storage.StateTransaction, expected octosql.Value) {
+	val, err := aggr.GetValue(ctx, tx)
+	assert.NotNil(t, val)
+	assert.Equal(t, expected.GetType(), val.GetType())
+
+	switch expected.GetType() {
+	case octosql.TypeInt:
+		assert.Equal(t, expected.AsInt(), val.AsInt())
+	case octosql.TypeFloat:
+		assert.Greater(t, expected.AsFloat(), val.AsFloat()-octosql.MakeFloat(eps).AsFloat())
+		assert.Less(t, expected.AsFloat(), val.AsFloat()+octosql.MakeFloat(eps).AsFloat())
+		//assert.Equal(t, expected.AsFloat(), val.AsFloat())
```

Comment:
A tiny change: maybe you could just define eps as octosql.MakeFloat(epsilon_value).AsFloat() and then you wouldn't have to write it twice

Reply:
sure, why not